benchmarks/common/db.go

```diff
@@ -86,20 +87,21 @@ func DropCreateRaw(db *sql.DB) {
 // RunScripts function execute all sql queries in scripts variable using using db
 func RunScripts(scripts []string, db *sql.DB) {
 	for _, script := range scripts {
-		fmt.Println(script)
+		logrus.Debugln(script)
 		_, err := db.Exec(script)
 		if err != nil {
-			fmt.Printf("Error: on sql - %v\n", script)
+			logrus.Debugf("Error: on sql - %v\n", script)
 			panic(err)
 		}
 	}
 }
 
 // IsExistsData checks is exists table with name <tablename>
 func IsExistsData(tablename string, db *sql.DB) bool {
+	return false
```

Comment:
Sure we need this line?

Reply:
removed this function because better to always re-generate, because we use also in transparent mode and to be sure that data generated with fresh encryptor_config, new keys, etc. With such checks it should be manually checked and more likely to forget and test with wrong data.